CHANGELOG.md

```diff
@@ -7,25 +7,29 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Next release
 
-## [[v8.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v8.2.0...HEAD)] - 2020-xx-xx]
-- Revert #631. Add back manage security group flags. (by @ryanooi)
+## [[v9.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v9.0.0...HEAD)] - 2020-xx-xx]
+
+- Write your awesome change here (by @you)
+
+# History
```

Comment:
Can you please remove unnecessary "hisotory" sections from the changelog. https://github.com/terraform-aws-modules/terraform-aws-eks/pull/752/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL36  and https://github.com/terraform-aws-modules/terraform-aws-eks/pull/752/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL85